.github/workflows/android-monkey.yaml

```
@@ -73,16 +73,26 @@ jobs:
         run: |
           cmake --version
           ninja --version
-          gradle -Parm64 -Parm32 -Pfirebase assembleGoogleDebug
+          gradle -Parm64 -Parm32 -Px86 -Pfirebase assembleGoogleDebug
```

По-умолчанию должно всё собирать, что нужно.
```suggestion
          gradle -Pfirebase assembleGoogleDebug
```

---

Вот и собирает что нужно - arm. А x86 никому не нужен.

---

Поменял тут. x86 надо выпилить из fdroid apk. Просто бесполезно занимает место.